src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -685,6 +685,54 @@ def refineMesh(cls, meshrefinement, options):
         refineElementsCount = options['Refine number of elements']
         meshrefinement.refineAllElementsCubeStandard3d(refineElementsCount, refineElementsCount, refineElementsCount)
 
+    @classmethod
+    def defineFaceAnnotations(cls, region, options, annotationGroups):
+        """
+        Add face annotation groups from the highest dimension mesh.
+        Must have defined faces and added subelements for highest dimension groups.
+        :param region: Zinc region containing model.
+        :param options: Dict containing options. See getDefaultOptions().
+        :param annotationGroups: List of annotation groups for top-level elements.
+        New face annotation groups are appended to this list.
+        """
+        parameterSetName = options['Base parameter set']
+        isMouse = 'Mouse 1' in parameterSetName
+        isHuman = 'Human 1' in parameterSetName
+
+        # create fissure groups
+        fm = region.getFieldmodule()
+
+        if isHuman:
+            upperLeftGroup = getAnnotationGroupForTerm(annotationGroups, get_lung_term("upper lobe of left lung"))
+            lowerLeftGroup = getAnnotationGroupForTerm(annotationGroups, get_lung_term("lower lobe of left lung"))
+
+            mesh2d = fm.findMeshByDimension(2)
```

Comment:
Move this line above if isHuman:
Fails for mouse. Please test mouse runs, check your lung test passes.